### DISCUSSION TITLE
Customize Happy Face on Display.

### DISCUSSION THREAD

**Discussion by LayerCakeMakes** [devbis/z03mmc#84] [Ideas]
This is a minor feature request for sure, but it would be nice to be able to customize when the happy/unhappy face is shown.
I, like many others, use the LYWSD03MMC to monitor the dryness of my 3D printer filament, so the lower the humidity the better which currently always leads to a unhappy face. I would love to be able to change this.

**Comment by tercioglu** [comment]
I would like to switch it off completely. :)

**Comment by devbis** [comment]
This functionality has been  implemented in the version 1.1.0
